README.md

```diff
@@ -37,16 +50,28 @@ targets:
     schedule: "*/1 * * * *"
 ```
 
-#### Verify running containers before deploying harpoon.
+#### Launch using systemd
+Two systemd files are provided to allow for Harpoon to either be ran as a user or as root. The files are differentiated by .root and .user.
 
-```
-podman ps
+Ensure that the `config.yaml` is correctly defined in the systemd service file before attmepting to start the service.
 
-CONTAINER ID  IMAGE       COMMAND     CREATED     STATUS      PORTS       NAMES
+For root
 ```
+setenforce 0
+cp systemd/harpoon.root /etc/systemd/system/harpoon.service
```

Comment:
Maybe a note about why setenforce 0 and/or that this is temporary (with an issue?)